heft-plugins/heft-typescript-plugin/src/TypeScriptPlugin.ts

```diff
@@ -157,6 +165,11 @@ export async function loadTypeScriptConfigurationFileAsync(
             // When merging objects, arrays will be automatically appended
             inheritanceType: InheritanceType.merge
           }
+        },
+        jsonPathMetadata: {
+          '$.additionalModuleKindsToEmit.*.outFolderName': {
```

Comment:
There's a possibility that this breaks something compared to doing the resolving manually. Different slashes?

Reply:
If that breaks, then that would be a deeper issue with the resolution we're performing that should be resolved anyway. Given that we haven't had issues using this in the Jest plugin/config, I think we're good here.